zebra-chain/src/block/height.rs

```diff
@@ -144,6 +144,31 @@ impl ZcashDeserialize for Height {
     }
 }
 
+impl ZcashSerialize for Height {
```

Comment:
Note: `ZcashSerialize` is for consensus-critical serialization, but heights have a few different serialization formats. 

So we can't really `impl ZcashSerialize for Height` - we just serialize heights in the context of larger data structures.

Reply:
Thank you for pointing this out. I went ahead too much with the serialization. I was following `zcash_deserialize`, noticed that it uses little endian, and implemented the serialization accordingly. This was all redundand.